src/test/kotlin/gamemap/CellTest.kt

```diff
@@ -0,0 +1,53 @@
+import gamemap.Cell
+import gamemap.CellState
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContain
+import java.lang.IllegalArgumentException
+import java.lang.IllegalStateException
+
+class CellTest : BehaviorSpec({
+    given("a cell") {
+        `when`("is open") {
+            then("should not be able to open") {
+                shouldThrow<IllegalStateException> { Cell(state = CellState.Open).open() }
+                    .message.shouldContain("cannot open a open cell")
```

Comment:
```suggestion
                shouldThrow<IllegalStateException> { 
                    Cell(state = CellState.Open).open() 
                } shouldHaveMessage "cannot open a open cell" 
```

**shouldHaveMessage**도 있습니다. 😊